config/migrations/20221021094201-move-persons-to-public.sparql

```diff
@@ -0,0 +1,29 @@
+PREFIX person: <http://www.w3.org/ns/person#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+DELETE {
+  GRAPH ?g {
+    ?s ?p ?o .
+  }
+}
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?s ?p ?o .
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?s a person:Person .
+    ?s ?p ?o .
+  }
+  FILTER (?g != <http://mu.semte.ch/graphs/public>)
+  FILTER EXISTS {
```

Comment:
Out of curiosity, why the explicit wrapping in `FILTER EXISTS`?

Reply:
I don't think it's strictly necessary. In a previous iteration this migration came after the `move-static-triples-to-public`, so the `?mandataris mandaat:isBestuurlijkeAliasVan ?person` triple was no longer in `?g`, but in the public graph.

This is no longer necessary though, the triple should still be in the same graph.